lib/salus.rb

```diff
@@ -64,7 +64,7 @@ def scan(
         raise e if ENV['RUNNING_SALUS_TESTS']
 
         puts "Could not send Salus report: (#{e.class}: #{e.message})"
-        Bugsnag.notify(e)
+        Bugsnag.notify(e) if ENV['BUGSNAG_API_KEY']
```

Comment:
hmm, definitely fixes the warning. Can we aim for a centralized solution? Maybe create a wrapper method too replace all the `Bugsnag.notify(e) if ENV['BUGSNAG_API_KEY']` checks

```
def log_error(e)
  if ENV['BUGSNAG_API_KEY']
    Bugsnag.notify(e)
  else
    # log to a generic logger
  end
end
```

Reply:
added wrapper Salus.bugsnag_notify to wrap the code in the if branch in your example. 
The else branch code is not added because the original code logs messages differently depending on the situation (ex. @report.warn vs @report.error)